src/extension.ts

```diff
@@ -36,17 +36,18 @@ export function activate(context: vscode.ExtensionContext) {
 
 				let result = document.getText();
 				const commentSpaces = checkCommentSpaces(result);
+				const cwd = vscode.workspace.rootPath;
```

Comment:
`rootPath` is deprecated. Maybe use `const cwd = vscode.workspace.workspaceFolders?.at(0)?.uri?.fsPath;` instead?

Reply:
ok, I will fix this recently.